pyomo/gdp/plugins/multiple_bigm.py

```diff
@@ -201,7 +200,6 @@ class MultipleBigMTransformation(GDP_to_MIP_Transformation, _BigM_MixIn):
 
     def __init__(self):
         super().__init__(logger)
-        self.handlers[Suffix] = self._warn_for_active_suffix
```

Comment:
Are we sure we want to silently ignore Suffixes?

Reply:
I changed this to ignore `LocalVars`, warn about ignoring `BigM`, and yell for everything else.